nearlib/test/integration.test.js

```diff
@@ -2,16 +2,20 @@ const { Account, KeyPair, Near, InMemoryKeyStore } = require('../');
 const dev = require('../dev');
 const fs = require('fs');
 const aliceAccountName = 'alice.near';
+// every new account has this codehash
+const newAccountCodeHash = "GKot5hBsd81kMupNCXHaqbhv3huEbxAFMLnpcX2hniwn";
+const localStorageAccountIdKey = 'dev_near_user';
 let nearjs;
 let account;
 let keyStore;
 
 beforeAll(async () => {
     keyStore = new InMemoryKeyStore();
+    const localStorage = createMockLocalStorage();
     nearjs = await dev.connect({
         nodeUrl: 'http://localhost:3030',
         useDevAccount: true,
-        deps: { keyStore },
+        deps: { keyStore, localStorage },
```

Comment:
should it just be `storage` instead of `localStorage` if it's made pluggable? E.g. someone theoretically might want to use https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage

Reply:
yeah, that's a better name